scripts/polarion-test-case-upload.sh

```diff
@@ -0,0 +1,33 @@
+# requires betelgeuse pypi package
+
+TC_PATH='tests/foreman/api tests/foreman/cli tests/foreman/ui tests/foreman/longrun tests/foreman/sys tests/foreman/installer tests/foreman/rhai tests/foreman/endtoend tests/foreman/virtwho'
```

Comment:
Can we instead exclude the paths this should not be run against, or at least comment what we're avoiding here? Looks like all of `tests/foreman` could be included.

Reply:
Refactored.
it has no sense